packages/f-checkout/src/components/Selector.vue

```diff
@@ -48,85 +27,34 @@ export default {
             return this.serviceType === CHECKOUT_METHOD_DELIVERY
                 ? this.$t('labels.deliveryOrderMethod')
                 : this.$t('labels.collectionOrderMethod');
-        }
-    },
-
-    watch: {
-        'fulfillment.times' (newValue) {
-            const selected = newValue.find(t => t.selected);
-            if (selected) {
-                this.selectedTime = selected.from;
-            }
+        },
+
+        fulfillmentTimes () {
+            /*
```

Comment:
Could you change these comments to follow jsDoc style ones please (so they also sit above the method name too rather than inside the method)